Yona/Yona/en.lproj/Localizable.strings

```diff
@@ -6,6 +6,10 @@
   Copyright © 2016 Yona. All rights reserved.
 */
 
+//Fabric API key file
+"fabric-alert-title" = "Intialization Error";
```

Comment:
Please spell this as "Initialization error" (lowercase e for error).

Reply:
updated